tests-scan

```diff
@@ -399,6 +400,7 @@ def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp
                 (priority, changes) = prioritize(status, title, labels, baseline, context, number)
             if not update or update_status(api, revision, context, status, changes):
                 checkout_ref = ref
+                # NOTE: hopefully these two are usable (REMOVE ME)
```

Comment:
I think that if we cannot count of `git fetch origin master` or `git fetch origin rhel-8-3` we are just screwed. Or should I try to resolve SHA for this as well?

Reply:
These branches don't get force-pushed. We really have to rely on them.